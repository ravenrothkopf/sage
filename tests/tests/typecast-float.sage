# testing float type casting

def int main () {
    printfl(float("9.07"))
    printfl(float(10))
}
