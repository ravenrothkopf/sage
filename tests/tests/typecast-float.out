9.07
10
